api/src/main/java/org/openmrs/module/fhir2/providers/r3/LocationFhirResourceProvider.java

```diff
@@ -113,6 +113,9 @@ public IBundleProvider searchLocations(@OptionalParam(name = Location.SP_NAME) S
 	        @OptionalParam(name = Location.SP_PARTOF, chainWhitelist = { "", Location.SP_NAME, Location.SP_ADDRESS_CITY,
 	                Location.SP_ADDRESS_STATE, Location.SP_ADDRESS_COUNTRY,
 	                Location.SP_ADDRESS_POSTALCODE }, targetTypes = Location.class) ReferenceAndListParam parent,
+	        @OptionalParam(name = "below", chainWhitelist = { "", Location.SP_NAME, Location.SP_ADDRESS_CITY,
```

Comment:
So it sounds like @ibacher  this extra parameter should not be here, and this new functionality should just be part of the "SP_PARTOF" parameter above?  I was having a problem with that, see my follow on comment and screenshot in the test case below.

Reply:
The tl;dr is that if I change this to name="partof:below" any parameter passed in that way gets assigned to *both* the ReferenceAndListParam ancestor and ReferenceAndListParam parent.  

And if I remove the "below" parameter, I'm not quite sure how to parse the "parent" ReferenceAndListParam to tell if this should be a standard "partof" vs a "partof:below"